elastic-utils-lib/src/cmr/elastic_utils/config.clj

```diff
@@ -15,6 +16,14 @@
     "Token used for basic auth authentication with elastic."
     {:default (str "Basic " (b64/encode (.getBytes "echo-elasticsearch")))})
 
+(defconfig elastic-scroll-timeout 
+  "Timeout for ES scrolling"
+  {:default "10m"})
```

Comment:
I started reading more about the resources that are consumed by scrolling. I'm thinking we should lower the default to "5m". Five minutes still seems like plenty of time between calls so someone could still manually use the API.